lib/ExpensiMark.js

```diff
@@ -445,17 +445,13 @@ export default class ExpensiMark {
                         .split('\n');
 
                     // Wrap each string in the array with <blockquote> and </blockquote>
-                    // Define a named function to wrap each line with blockquote
-                    function wrapWithBlockquote(line) {
+                    resultString = resultString.map((line) => {
                         return `<blockquote>${line}</blockquote>`;
-                    }
-
-                    // Use _.map with the named function
-                    resultString = _.map(resultString, wrapWithBlockquote);
+                    });
 
-                    function processString(m) {
-                        // Recursive function to replace nested <blockquote> with ">"
-                        function replaceBlockquotes(text) {
```

Comment:
We don't need this function anymore right?
In this case, I think we can remove below comment:
```
 // Recursive function to replace nested <blockquote> with ">"
```